ouroboros-consensus-cardano/changelog.d/20240111_195320_facundo.dominguez_lightweight_checkpointing.md

```diff
@@ -0,0 +1,23 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+### Non-Breaking
```

Comment:
Should this be Breaking do to the changes to APIs in the code?

I initially chose based on whether this would break the behavior of honest nodes :)

Reply:
FTR: This should indeed be based on the changes to the code based on PVP :+1:

Reply:
I changed this to the Breaking category.